src/style.css

```diff
@@ -0,0 +1,51 @@
+* {
+  margin: 0 px;
+  padding: 0 px;
+  border: 0 px;
+  box-sizing: 0 px;
+}
+
+.container {
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: flex-start;
+}
+
+.row-1,
+.row-2,
+.row-3,
+.row-4,
+.row-5,
+.row-6 {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.stars__star-1,
+.stars__star-2,
+.stars__star-3,
+.stars__star-4,
+.stars__star-5 {
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  border-radius: 0.5px;
+
+}
+
+.stars__star-6,
+.stars__star-7,
+.stars__star-8,
+.stars__star-9,
+.stars__star-10 {
```

Comment:
You need to make a class with the stars amount and use nth-child to set active stars. If, for example, you need 100 lines of stars, then you will significantly inflate the code with a large number of classes